modules/social_features/social_user_export/social_user_export.module

```diff
@@ -189,6 +189,26 @@ function social_user_export_groups_count(UserInterface $account) {
     ->fetchField();
 }
 
+/**
+ * Returns the quantity of likes by a user.
+ *
+ * @param \Drupal\user\UserInterface $account
+ *   The account to get the data for.
+ *
+ * @return int
+ *   number of likes
+ */
+function social_user_export_likes_count(UserInterface $account) {
+  $query = \Drupal::database()->select('votingapi_vote', 'v');
+  $query->addField('v', 'type');
```

Comment:
I think you don't need this addField since in the end you will count the results. You can see the other export queries where they also don't use the addField.